aten/src/ATen/native/cuda/PowKernel.cu

```diff
@@ -168,16 +168,6 @@ void pow_tensor_scalar_kernel_impl(TensorIteratorBase& iter,
 }
 
 void pow_tensor_scalar_kernel(TensorIteratorBase& iter, const Scalar& exp_scalar) {
```

Comment:
This removes the optimization from cpu-scalar exponents:
https://github.com/pytorch/pytorch/blob/014f91a9d9f94ac9a7f0711600240d7cd7f69844/aten/src/ATen/native/cuda/PowKernel.cu#L134

Reply:
The only way to get there is through `pow_Tensor_Scalar_out` -> `pow_tensor_scalar_stub`, which is where we put all the optimisations.

Reply:
The line I linked is in `pow_tensor_tensor_kernel`.